src/pages/trash.js

```diff
@@ -0,0 +1,34 @@
+import { Sidebar } from "../components/allComp";
+import { useTrash } from "../context/trashContext";
+export const TrashPage = () => {
+    const { noteTrash, RestoreTrashNote, DeleteNoteFromTrash } = useTrash();
+    return (
+        <div>
+            <Sidebar />
```

Comment:
If you want you can fix the sidebar directly into the routes path or home page

Reply:
But the sidebar is not in the home page, thats why I kept it here